config/services/kubelet-readonly.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<service>
+  <short>Kubernetes Kubelet read</short>
+  <description>The kubelet API is used to communicate between kube-scheduler and the node.</description>
+  <port protocol="tcp" port="10255"/>
```

Comment:
Does it really makes sense to separate `kubelet` and `kubelet-readonly`? Would a user ever want to keep the readonly port closed? If no, then please combine this with `kubelet`.

Reply:
I was thinking more of enabling only the readonly port in a certian zone but I don't really know how much this is used.

Reply:
Okay. We'll leave it as-is since it was the existing behavior. In the future we could `include` the read-only service from the `kubelet` service.